Various Sitewide Fixes & Improvements. (#266)
FabricMC/fabric-docs: README.md

## Contributing

Please refer to the [contribution guidelines](./contributing) for more information.

If you wish to contribute translations, you can do so via the [Crowdin project](https://crowdin.com/project/fabricmc).

## Attribution & Credits

> - Thank you to all the contributors that have helped create guides for the documentation! Also a huge thank you to users who have helped translate existing guides into other languages via the [Crowdin project](https://crowdin.com/project/fabricmc).
> - `CountryFlagsPolyfill.woff2` is adapted from Twemoji Mozilla by Mozilla & X (formerly Twitter) contributors, licensed under CC-BY-4.0. Modifications include storing only the flag emoticons ("Emojis"). Original license details are available at [Twemoji Mozilla](https://github.com/mozilla/twemoji-colr/blob/master/LICENSE#license-for-the-visual-design).

Reviewer: ```suggestion
- Thank you to all the [contributors to the repo](https://github.com/FabricMC/fabric-docs/graphs/contributors), who are helping create guides for the documentation!
- A huge thank you to users who are translating guides to other languages via the [Crowdin project](https://crowdin.com/project/fabricmc)
- `CountryFlagsPolyfill.woff2` is adapted from "Twemoji Mozilla" by Mozilla & X (formerly Twitter) contributors, licensed under CC-BY-4.0. Modifications include storing only the flag emoticons ("Emojis"). Original license details are available at [Twemoji Mozilla](https://github.com/mozilla/twemoji-colr/blob/master/LICENSE#license-for-the-visual-design)
```
